hack/build.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# This script builds go components.
+# You can set the platform to build with BUILD_PLATFORMS, with format: `<os>/<arch>`
+# And binaries will be put in `_output/<os>/<arch>/`
+#
+# Usage:
+#   hack/build.sh <target>
+# Args:
+#   $1:              target to build
+#   BUILD_PLATFORMS: platforms to build. You can set one or more platforms separated by comma.
```

Comment:
`BUILD_PLATFORMS` belongs to `Environments` not  `Args`, right?

Reply:
accurately, it is Environments